api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove all classes that are not explicitly needed.  The SDK creates a bunch of "helpful" base classes for a common use case, but these should be deleted before committing if they are not needed.